src/node.js

```diff
@@ -317,12 +317,7 @@
       if (microtasksScheduled)
         return;
 
-      nextTickQueue.push({
-        callback: runMicrotasksCallback,
-        domain: null
-      });
-
-      tickInfo[kLength]++;
+      process.nextTick(runMicrotasksCallback);
```

Comment:
Maybe it's better to use `nextTick` instead of `process.nextTick`? don't like when we rely on globals being untouched